tests/test_meshtastic_utils_connect_paths.py

```
@@ -23,6 +24,14 @@
 from mmrelay.meshtastic_utils import connect_meshtastic, on_lost_meshtastic_connection
 
 
+def _schedule_reconnect_closing_coro(coro, loop=None):
```

Ruff (0.15.10)

[warning] Missing return type annotation for private function `_schedule_reconnect_closing_coro`

(ANN202)